src/simulation/AliasManager.java

```diff
@@ -10,6 +10,7 @@
     private Map<String, String> aliases;
     private final static String ALIAS_HEADER = "Alias";
     private final static String FORMULA_HEADER = "Formula";
+    private final static String BACKTICK = "`";
```

Comment:
A name like `NESTED_ALIAS_WRAPPER_CHAR` would work better since it would allow us to change the value in the future without needing to change the name.